Dockerfile

```diff
@@ -18,9 +20,16 @@ COPY pkg/ pkg/
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
 # Use ubi-minimal as minimal base image to package the manager binary
-# Refer to https://catalog.redhat.com/software/containers/ubi8-minimal/5c64772edd19c77a158ea216 for more details
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.2-691
+# For more details and updates, refer to
+# https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3
 COPY --from=builder /opt/app-root/src/manager /
 USER 65532:65532
 
+LABEL description="RHTAP Build Service operator"
+LABEL io.k8s.description="RHTAP Build Service operator"
+LABEL io.k8s.display-name="build-service-operator"
+LABEL io.openshift.tags="rhtap"
+LABEL summary="RHTAP Build Service"
```

Comment:
Could the labels be in a separate commit?

Reply:
ok